libraries/AC_WPNav/AC_WPNav.h

```diff
@@ -220,6 +220,17 @@ class AC_WPNav
     /// return the crosstrack_error - horizontal error of the actual position vs the desired position
     float crosstrack_error() const { return _pos_control.crosstrack_error();}
 
+    ///
+    /// offset methods
+    ///
+    /// add an offset to vehicle's target position in NED meters
+    /// only xy offsets are supported
+    void set_pos_offset(const Vector3p& pos_offset_NED);
```

Comment:
I wonder if these could be just in the position controller and work in any mode then.

Reply:
I've moved the offset accessors to the position controller